src/main/java/org/truffleruby/language/methods/SharedMethodInfo.java

```diff
@@ -34,7 +34,7 @@ public final class SharedMethodInfo {
     private final Arity arity;
     /** The original name of the method. Does not change when aliased. Looks like "block in foo" or "block (2 levels) in
      * foo" for blocks. This is the name shown in backtraces: "from FILE:LINE:in `NAME'". */
-    private final String backtraceName;
+    private final String originalName;
```

Comment:
I appreciate the intention but I think it is more misleading than the previous field name.
For instance if someone uses `myProc.sharedMethodInfo.originalName`, do they really expect `block (2 levels) in foo` as the "original name" ? I would expect `foo` for a field named "originalName".

I think `InternalMethod#getOriginalName()` and that using `SharedMethodInfo#getMethodName()` is better and achieves the same clarity.

Reply:
Also SharedMethodInfo is basically "static information from the parser", so there is no concept of original or alias here, those are runtime concepts.

Reply:
I discussed it with Kevin, the old name is not necessarily clearer, let's try this new one.